server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -1,42 +1,47 @@
 import { SentReferral, ServiceCategory } from '../../services/interventionsService'
 import CalendarDay from '../../utils/calendarDay'
-import PresenterUtils from '../../utils/presenterUtils'
+import PresenterUtils, { SortableTableHeaders, SortableTableRow } from '../../utils/presenterUtils'
 import utils from '../../utils/utils'
 
 export default class DashboardPresenter {
   constructor(private readonly referrals: SentReferral[], private readonly serviceCategories: ServiceCategory[]) {}
 
-  readonly tableHeadings = ['Date received', 'Referral', 'Service user', 'Intervention type', 'Caseworker', 'Action']
+  readonly tableHeadings: SortableTableHeaders = [
+    { text: 'Date received', sort: 'none' },
+    { text: 'Referral', sort: 'none' },
+    { text: 'Service user', sort: 'none' },
+    { text: 'Intervention type', sort: 'none' },
+    { text: 'Caseworker', sort: 'none' },
+    { text: 'Action', sort: 'none' },
+  ]
 
-  readonly tableRows: { text: string; sortValue: string | null; href: string | null }[][] = this.referrals.map(
```

Comment:
the shorter line length here means the `referral => {` is now on the same line and all the indentation below is affected. the code here isn't changed at all apart from the method signature.